pulsar-broker/src/test/java/org/apache/pulsar/client/impl/NegativeAcksTest.java

```diff
@@ -106,6 +106,9 @@ public void testNegativeAcks(boolean batching, boolean usePartitions, Subscripti
         log.info("Test negative acks batching={} partitions={} subType={} negAckDelayMs={}", batching, usePartitions,
                 subscriptionType, negAcksDelayMillis);
         String topic = BrokerTestUtil.newUniqueName("testNegativeAcks");
+        if (usePartitions) {
+            admin.topics().createPartitionedTopic(topic, 2);
```

Comment:
Question: 
I'm not sure if the topics that have been created before are cleaned up every time we run the tests? Will this affect the accuracy of the test?

Reply:
`newUniqueName ` will use UUID to create topic.

Reply:
Oh, thanks.